keras_nlp/models/whisper/whisper_tokenizer.py

```diff
@@ -149,51 +146,3 @@ def get_config(self):
     @classproperty
     def presets(cls):
```

Comment:
curious...other models like GPT2, BERT have `from_preset` methods, don't we have need it here?

Reply:
Should actually come int automatically from the parent class I think!

Reply:
+1